packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1365,6 +1365,40 @@ export const FontFamilyNotFound = {
 	hint: 'This is often caused by a typo. Check that your Font component is using a `cssVariable` specified in your config.',
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @message No parser found for file `example.md`. Pass a parser to the `file` loader (e.g. `parser: mdx`).
+ * @see
+ *  - [parsers](https://docs.astro.build/en/guides/imports/#parsers)
+ * @description
+ * Thrown when the `file` loader can’t determine which parser to use.
+ */
+export const FileParserNotFound = {
+	name: 'FileParserNotFound',
+	title: 'File parser not found',
+	message: (fileName: string) =>
+		`No parser was found for '${fileName}'. Pass a parser function (e.g. \`parser: mdx\`) to the \`file\` loader.`,
+	hint:
+		'See https://docs.astro.build/en/guides/imports/#parsers for the list of available parser options.',
```

Comment:
This is a list of supported file types, but I am not sure that this is a list of parser options that someone can pass to the file loader? (It doesn't tell them the exact name to use? And doesn't mention parsing/content collections at all?) I don't think we actually give anyone a list of parser options anywhere?

Reply:
thanks for catching that. The imports/#parsers link only lists built-in file types, not the actual `parser` names a user can pass. 

so It's better to I’ll remove it instead.

Reply:
@sarah11918 
delete this hint `684c028`